pkg/app/pipedv1/apistore/applicationstore/store.go

```diff
@@ -36,6 +36,8 @@ type Lister interface {
 	ListByPlatformProvider(name string) []*model.Application
 	// Get retrieves a specifiec deployment for the given id.
 	Get(id string) (*model.Application, bool)
+	// ListByPluginName lists all applications for a given plugin name.
+	ListByPluginName(name string) []*model.Application
```

Comment:
Should we replace the ListByPlatformProver?

Reply:
@khanhtc1202 Your right. We don't need it for now. So I removed it. [374de36](https://github.com/pipe-cd/pipecd/pull/5457/commits/374de369533d577608e3d070044cccb9b9b16348)